plugins/Terminal/js/actions/commandline.js

```diff
@@ -0,0 +1,47 @@
+import * as constants from '../constants/commandline.js'
+
+export const addCommand = (command) => ({
+	type: constants.ADD_COMMAND,
+	command,
+})
+export const updateCommand = (command, id, dataChunk, stat) => ({
+	type: constants.UPDATE_COMMAND,
+	command, id, stat,
+	dataChunk,
+})
+
+
+export const loadPrevCommand = () => ({
+	type: constants.LOAD_PREV_COMMAND,
+})
+export const loadNextCommand = () => ({
+	type: constants.LOAD_NEXT_COMMAND,
+})
+export const setCurrentCommand = (commandText) => ({
+	type: constants.SET_CURRENT_COMMAND,
+	command: commandText,
+})
+
+
+export const showWalletPrompt = () => ({
+	type: constants.SHOW_WALLET_PROMPT,
+})
+export const hideWalletPrompt = () => ({
+	type: constants.HIDE_WALLET_PROMPT,
+})
+
+
+export const showSeedPrompt = () => ({
+	type: constants.SHOW_SEED_PROMPT,
+})
+export const hideSeedPrompt = () => ({
+	type: constants.HIDE_SEED_PROMPT,
+})
```

Comment:
hmm. I'm not sure I like `toggle`; it subtly binds your action to the current state of the program. That is, you need to know what state the list is in when you toggle it. Seems safer to explicitly `show` or `hide` it. (For example, what if two concurrent actions both try to toggle an element?)